backend/services/create_team.py

```diff
@@ -0,0 +1,52 @@
+from battle.battles.battle import validate_sum_pokemons, verify_pokemon_is_saved
+
+from pokemon.helpers import verify_pokemon_exists_api
+
+
+def verify_if_data_is_valid(data):  # pylint: disable=too-many-return-statements
+    message_error = None
+
+    if 'battle' not in data:
+        message_error = 'ERROR: Select a valid battle'
+        return message_error
```

Comment:
This may be required on the form, but not on the serializer since you can set a field to be required

Reply:
hmm great!